.github/workflows/build_prs.yaml

```
@@ -20,4 +20,10 @@ jobs:
       - run: yarn install --frozen-lockfile --ignore-scripts
       - run: yarn lint
       - run: yarn build:backend
-      #- run: yarn build:frontend # not needed
+      - run: yarn test:report
+      - name: Upload test report to build # Done this way since external PRs would not be able to write the check. See https://github.com/marketplace/actions/test-reporter#recommended-setup-for-public-repositories
+        uses: actions/upload-artifact@v2
+        if: success() || failure()
```

Here you could use
```suggestion
        if: always()
```
But this is also fine IMO